GoogleMapsComponents/GoogleMapsComponents.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
It should stay netstandart2.0. At most you could increase to 2.1

Reply:
I was going for the js isolation.  I thought that wasn't included in 2.1.
Regardless.  The js isolation was just a "nice-to-have" that I thought I'd include in this PR.  The eval in the tryParseJson along with the ImageMapType are the functional changes.